app/routes/auth.ts

```diff
@@ -1,11 +1,13 @@
 import express from 'express';
+import { AuthController } from '../http/controllers/AuthController';
 import { IndexController } from '../http/controllers/IndexController';
 
 const router = express.Router();
 
 import AuthMiddleware from '../http/middleware/AuthMiddleware';
 // Everywhere below here will require authentication.
-router.use(AuthMiddleware);
+
+router.post('/login', AuthController.login);
```

Comment:
Missing input validation. 

Use the zod library and the provided validation middleware. There should be examples in this project, as well as the ldap api, or go bother jung.